Microsoft.NET.Build.Containers/Constants.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Reflection;
+
+namespace Microsoft.NET.Build.Containers;
+
+public static class Constants {
```

Comment:
Just wondering whether this can live together with `KnownStrings` (but `Constants` seems a better term).